C/services/south/south.cpp

```diff
@@ -815,7 +834,7 @@ void SouthService::restart()
 /**
  * Configuration change notification
  */
-void SouthService::configChange(const string& categoryName, const string& category)
+void SouthService::configChangeReal(const string& categoryName, const string& category)
```

Comment:
I really do not like the naming of this.

Reply:
Changed.